lib/ruumba.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Can we ignore in `.rubocop.yml` rather than applying to each file?

Reply:
Done